sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnosticsContext.java

```diff
@@ -311,7 +311,12 @@ void addDiagnostics(CosmosDiagnostics cosmosDiagnostics) {
         }
     }
 
-    Collection<ClientSideRequestStatistics> getDistinctCombinedClientSideRequestStatistics() {
+    // todo (abhmohanty): revert public API change - used to hook into to track 404/1002s for operation
```

Comment:
Just as another way to track the todo before merge

Reply:
Fixed in next iteration.